src/components/Header.tsx

```diff
@@ -0,0 +1,35 @@
+import Slider from 'react-slick'
+import 'slick-carousel/slick/slick.css'
+import 'slick-carousel/slick/slick-theme.css'
+import { adSlogans } from '~/data/adSlogans'
+
+const Header = () => {
+	const settings = {
+		accessibility: false,
+		adaptiveHeight: true,
+		arrows: false,
+		autoplaySpeed: 5000,
+		infinite: true,
+		speed: 1500,
+		slidesToShow: 1,
+		slidesToScroll: 1,
+		autoplay: true,
+	}
```

Comment:
Pls move it outside. This way React create a new object in every rerender

Reply:
resolved in header branch